core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -442,6 +462,8 @@ void print_stack_arm32(int level, struct unwind_state_arm32 *state,
 	} while (unwind_stack_arm32(state, exidx, exidx_sz, stack, stack_size));
 }
 
+#endif	/* CFG_CORE_SYMS */
```

Comment:
style: replace tabulation with a space character.
same comment in `unwind_arm64.c`.